src/lib/components/forms/form/form-components/checkbox/checkbox.component.spec.ts

```diff
@@ -1,24 +1,134 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { CheckboxComponent } from './checkbox.component';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { MatCheckboxHarness } from '@angular/material/checkbox/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MockTooltipDirective } from '../../../../../testing/mock-tooltip.directive';
+import { By } from '@angular/platform-browser';
+import { TerraPlacementEnum } from '../../../../../helpers';
 
+// tslint:disable-next-line:max-function-line-count
 describe('CheckboxComponent', () => {
     let component: CheckboxComponent;
     let fixture: ComponentFixture<CheckboxComponent>;
+    let loader: HarnessLoader;
+    let checkbox: MatCheckboxHarness;
 
     beforeEach(async () => {
-        await TestBed.configureTestingModule({
-            declarations: [CheckboxComponent]
-        }).compileComponents();
-    });
+        TestBed.configureTestingModule({
+            imports: [MatCheckboxModule],
```

Comment:
Da vermutlich ngModel verwendet wird, musst du noch das FormsModule importieren
```suggestion
            imports: [MatCheckboxModule, FormsModule],
```